### ISSUE TITLE
[4.0] a11y toggles

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#15678]
_Ignoring any comment about the toggle switches in aurora being good or bad design ideas._ 

They are currently not accessible as there is no active or focus outline. You can easily see this in the global configuration by tabbing between fields. All text fields have a focus blue outline but the toggles do not.

Also for reference https://inclusive-components.design/toggle-button/
